src/getting_grain.md

```diff
@@ -2,16 +2,56 @@
 title: Getting Grain
 ---
 
-## Building Grain
+We have a couple different ways to get Grain. Most developers will prefer the Packaged Grain binaries, but make sure to read the note about speed!
 
-Grain currently only works on Mac and Linux. We are hard at work trying to integrate Binaryen and OCaml on Windows, but this is uncharted territory! Please let us know if you have this low-level expertise.
+## Packaged Grain
 
-To build the compiler, you'll need [Node.js](https://nodejs.org/en/download/current/) v14 and [Yarn](https://yarnpkg.com/getting-started/install).
+The Grain toolchain (including our CLI, compiler, runtime, and standard library) is shipped as a single binary. Binaries are available for [MacOS x64](#MacOS-x64---Homebrew), [Linux x64](#Linux-x64---Download), and [Windows x64](#Windows-x64---Download).
+
+__Note:__ These binaries are a bit slow the first time you build a project (around 70 seconds) because they build and output the runtime & standard library into your project. If you need extra speed, you can build the native compiler from source! See [Building Grain from Source](#Building-Grain-from-Source) below.
+
+### MacOS x64 - Homebrew
+
+The easiest way to install on MacOS is to install from our cask using [homebrew](https://brew.sh).
+
+```sh
+brew install --no-quarantine --cask grain-lang/tap/grain
+```
+
+_The `--no-quarantine` flag will avoid having to approve the binary in the Security Center._
+
+### MacOS x64 - Download
+
+If you'd prefer not to use homebrew, you can [download it](https://github.com/grain-lang/grain/releases/download/grain-v0.3.0/grain-mac-x64) directly from GitHub or using `curl`.
+
+```sh
+sudo curl https://github.com/grain-lang/grain/releases/download/grain-v0.3.0/grain-mac-x64 -L --output /usr/local/bin/grain && sudo chmod +x /usr/local/bin/grain
+```
+
+### Linux x64 - Download
+
+You can [download it](https://github.com/grain-lang/grain/releases/download/grain-v0.3.0/grain-linux-x64) directly from GitHub or using `curl`.
+
+```sh
+sudo curl https://github.com/grain-lang/grain/releases/download/grain-v0.3.0/grain-linux-x64 -L --output /usr/local/bin/grain && sudo chmod +x /usr/local/bin/grain
+```
+
+### Windows x64 - Download
+
+You can [download it](https://github.com/grain-lang/grain/releases/download/grain-v0.3.0/grain-win-x64.exe) directly from GitHub.
+
+You'll either want to put it into your path or keep it inside your project and invoke with `./grain-win-x64.exe`.
+
+## Building Grain from Source
+
+To get access to the entirely native compiler, you can build Grain from source.
```

Comment:
This implies that packaged isn't native, but we didn't say that explicitly. Should we say something about it? I could go either way.

Reply:
Yeah, I think we could mention that. Let me see if I can come up with a good way to phrase it.

Reply:
See above?